test/unit/gateways/firstdata_e4_test.rb

```diff
@@ -163,8 +163,76 @@ def test_add_swipe_data_with_creditcard
     end.respond_with(successful_purchase_response)
   end
 
+  def test_supports_scrubbing?
+    assert @gateway.supports_scrubbing?
+  end
+
+  def test_scrub
+    assert_equal @gateway.scrub(pre_scrubbed), post_scrubbed
+  end
+
   private
 
+  def pre_scrubbed
+    <<-PRE_SCRUBBED
+      opening connection to api.demo.globalgatewaye4.firstdata.com:443...
+      opened
+      starting SSL for api.demo.globalgatewaye4.firstdata.com:443...
+      SSL established
+      <- "POST /transaction/v11 HTTP/1.1\r\nContent-Type: application/xml\r\nAccepts: application/xml\r\nAccept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3\r\nAccept: */*\r\nUser-Agent: Ruby\r\nConnection: close\r\nHost: api.demo.globalgatewaye4.firstdata.com\r\nContent-Length: 593\r\n\r\n"
+      <- "<?xml version=\"1.0\" encoding=\"UTF-8\"?><Transaction><ExactID>REDACTED</ExactID><Password>REDACTED</Password><Transaction_Type>00</Transaction_Type><DollarAmount>1.00</DollarAmount><Card_Number>4242424242424242</Card_Number><Expiry_Date>0916</Expiry_Date><CardHoldersName>Longbob Longsen</CardHoldersName><CardType>Visa</CardType><VerificationStr1>1234 My Street|K1C2N6|Ottawa|ON|CA</VerificationStr1><CVD_Presence_Ind>1</CVD_Presence_Ind><VerificationStr2>123</VerificationStr2><Reference_No>1</Reference_No><Reference_3>Store Purchase</Reference_3><CAVV/><XID/><Ecommerce_Flag/></Transaction>"
```

Comment:
There are no credentials in the fixtures file, so I needed to run this with credentials that I can't leave in the code publicly.